### PR TITLE
add leticiarosario

### LINKED ISSUE #554: Leticia Rosario
leticiarosario

### PR DESCRIPTION
**Pull request by Leticiarosario**
resolve #554

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -995,4 +995,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
   }
```
Comment (starlightknown) [inline data.json:997]:
```suggestion
  },
```

**Review by starlightknown — CHANGES_REQUESTED** [review]
Other than this small change, it's good. Also please close the other PR #555

File: data.json (hunk 1 above)
Comment (schmelto) [reply data.json:997]: With this change it looking good for me

**Comment by schmelto** [comment]
duplicate with #555 and #557

**Review by MRslimcoder — APPROVED** [review]
LGTM

File: data.json (hunk 1 above)
Comment (schmelto) [inline data.json:997]:
```suggestion
   },
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 2]
```diff
@@ -994,5 +994,11 @@
     "githubUsername": "bevennyamande",
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
+   },
```
Comment (schmelto) [inline data.json:997]:
```suggestion
  },
```

**Review by schmelto — APPROVED** [review]

**Pull request merged**